tests/integration/test_integration_all_rust.py

```diff
@@ -595,229 +590,259 @@ class TestRustWebPush(unittest.TestCase):
         "sub": "mailto:admin@example.com",
     }
 
+    @staticmethod
+    def clear_sentry_queue():
+        """Clear any values present in Sentry queue."""
+        while not MOCK_SENTRY_QUEUE.empty():
+            MOCK_SENTRY_QUEUE.get_nowait()
+        return
+
     def tearDown(self):
         """Tear down and log processing."""
         process_logs(self)
-        while not MOCK_SENTRY_QUEUE.empty():
-            MOCK_SENTRY_QUEUE.get_nowait()
+        TestRustWebPush.clear_sentry_queue()
 
     def host_endpoint(self, client):
         """Return host endpoint."""
         parsed = urlparse(list(client.channels.values())[0])
-        return "{}://{}".format(parsed.scheme, parsed.netloc)
+        return f"{parsed.scheme}://{parsed.netloc}"
 
-    @inlineCallbacks
-    def quick_register(self):
+    @pytest.mark.asyncio
```

Comment:
regardless of asyncio auto mode or not, this and `shut_down` shouldn't be decorated w/ `@pytest.mark.asyncio` as they're not tests

Reply:
Thanks, good catch 👍